src/index.html

```diff
@@ -6,9 +6,67 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
```

Comment:
please fix the code styling according to the checklist
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/582a213b-db4d-424d-bcbe-25805ef4109b)